client/my-sites/stats/features/modules/stats-authors/stats-authors.tsx

```diff
@@ -0,0 +1,90 @@
+import { StatsCard } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { postAuthor } from '@wordpress/icons'; // TODO this isn't quite the right icon. Need to update when we can locate the correct icon
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import QuerySiteStats from 'calypso/components/data/query-site-stats';
+import { useSelector } from 'calypso/state';
+import {
+	isRequestingSiteStatsForQuery,
+	getSiteStatsNormalizedData,
+} from 'calypso/state/stats/lists/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import EmptyModuleCard from '../../../components/empty-module-card/empty-module-card';
+import { SUPPORT_URL } from '../../../const';
+import StatsModule from '../../../stats-module';
+import StatsModulePlaceholder from '../../../stats-module/placeholder';
+
+type StatClicksProps = {
+	className?: string;
+	period: string;
+	query: {
+		date: string;
+		period: string;
+	};
+	moduleStrings: {
+		title: string;
+		item: string;
+		value: string;
+		empty: string;
+	};
+};
+
+const StatClicks: React.FC< StatClicksProps > = ( { period, query, moduleStrings, className } ) => {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId ) as number;
+	const statType = 'statsTopAuthors';
+
+	const requesting = useSelector( ( state ) =>
+		isRequestingSiteStatsForQuery( state, siteId, statType, query )
+	);
+	const data = useSelector( ( state ) =>
+		getSiteStatsNormalizedData( state, siteId, statType, query )
+	) as [ id: number, label: string ];
+
+	return (
+		<>
+			{ siteId && statType && (
+				<QuerySiteStats statType={ statType } siteId={ siteId } query={ query } />
+			) }
+			{ requesting && <StatsModulePlaceholder isLoading={ requesting } /> }
+			{ ( ! data || ! data?.length ) && (
+				<StatsCard
+					className={ className }
```

Comment:
Let's add `stats-card--empty-variant` (https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/stats/features/modules/stats-referrers/stats-referrers.tsx#L61) to help with vertical centering.